README.md

```diff
@@ -44,8 +44,10 @@ var feature = parser.Parse("Feature: ...");
 ```ruby
 # Ruby
 require 'gherkin3/parser'
+require 'gherkin3/token_scanner'
+
 parser = Gherkin3::Parser.new
-feature = parser.parse("Feature: ...")
+feature = parser.parse(Gherkin3::TokenScanner.new("Feature: ..."))
```

Comment:
Done. I sent you another PR. Now `TokenScanner` is able to extract io from another `TokenScanner`-instance to make it backward compatible.